lib/pengine/pe_output.c

```diff
@@ -1886,6 +1886,28 @@ node_text(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
+/*!
+ * \internal
+ * \brief Convert an integer health value to a string representation
+ *
+ * \param[in] health  Integer health value
+ *
+ * \retval \c PCMK_VALUE_RED if \p health is less than 0
+ * \retval \c PCMK_VALUE_YELLOW if \p health is equal to 0
+ * \retval \c PCMK_VALUE_GREEN if \p health is greater than 0
+ */
+static const char *
+health_str(int health)
```

Comment:
This makes me remember that we recently decided to with `_text` for such functions, but then I just realized that `_text` is also used for text output implementations. I guess it's potentially confusing either way.

Reply:
Thanks, I created these commits before that decision. I already went back and renamed another one of these helpers (I think `no_quorum_policy_text()`) but I missed this one.